cmd/sos_show.go

```diff
@@ -0,0 +1,197 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/exoscale/cli/table"
+	"github.com/spf13/cobra"
+)
+
+type sosACLShowOutput struct {
+	Access string `json:"access"`
+	Value  string `json:"value"`
+}
+
+type sosMetadataShowOutput struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+type sosHeadersShowOutput struct {
+	Key   string `json:"key"`
+	Value string `json:"value"`
+}
+
+type sosShowOutput struct {
+	URL      string                  `json:"url"`
+	ACL      []sosACLShowOutput      `json:"acl"`
+	Metadata []sosMetadataShowOutput `json:"metadata"`
+	Headers  []sosHeadersShowOutput  `json:"headers"`
+}
+
+func (o *sosShowOutput) toJSON() { outputJSON(o) }
+
+func (o *sosShowOutput) toText() { outputText(o) }
+
+func (o *sosShowOutput) toTable() {
+	t := table.NewTable(os.Stdout)
+
+	if o.ACL != nil {
+		buf := bytes.NewBuffer(nil)
+		at := table.NewEmbeddedTable(buf)
+
+		switch {
+		case o.ACL[0].Access == "Canned":
```

Comment:
I was thinking of an even simpler display for canned ACLs, something like:

```
./exo sos show xxxx-xxxx IMG_1184
┼─────────┼────────────────────────────────────────────────────┼
│ ACL     │ public-read                                        │
│         │                                                    │
│ Headers │       KEY          VALUE                           │
│         │   content-type   image/jpeg                        │
│         │                                                    │
│ URL     │ https://sos-de-fra-1.exo.io/xxxx-xxxx/IMG_1184.jpg │
┼─────────┼────────────────────────────────────────────────────┼
```

WDYT?

Reply:
I think it's good to tell him what it is, but, it's true that on your example it's more readable